src/components/organisms/Badges/Badges.tsx

```diff
@@ -84,29 +115,29 @@ export const Badges: React.FC<{
 
         return {
           venue,
-          image: venue.mapIconImageUrl,
+          image: venue?.host?.icon,
```

Comment:
Do we want a default set on this?

Reply:
Not sure what the default would be.

Reply:
This is handled in the `BadgeImage` something like image ?? defaultAvatar